cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -926,6 +926,43 @@ action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Sysmon
 
 
+[Windows - Firewall is Disabled]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 2 * * * *
+description = The alert will be triggered if the Windows Firewall has been disabled from its active state in the past 24 hours. \
```

Comment:
description = This alert will be triggered when a Windows Firewall on the host is disabled. We suppress the alert for the host for 24 hours before triggering it again.

Alert should execute every hour.
Alert should run for -62m@m to -2m@m

**How to do suppression**:
* Search for the results in index=cyences (see Overview dashboard for macro usage) AND alert_name="<this alert>" and see if the same host appear then ignore the row.

Reply:
As per the discussion, added the inline comments and updated the earliest time to @d